### PR TITLE
fix: remove license warning for customers

### PR DESCRIPTION
**Pull request by ericpaulsen**
this PR removes the license warning for customers - this will only apply to trial licenses. 

current state:

image

### CODE REVIEW

**Review by kylecarbs — APPROVED** [review]

File: enterprise/coderd/license/license.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func Entitlements(
 		if isTrial {
 			showWarningDays = 7
 		}
-		isExpiringSoon := daysToExpire > 0 && daysToExpire < showWarningDays
+		isExpiringSoon := isTrial && daysToExpire > 0 && daysToExpire < showWarningDays
```
Comment (coadler) [inline enterprise/coderd/license/license.go:90]: This is probably dangerous since it won't show to admins if the license is expiring

Comment (coadler) [reply enterprise/coderd/license/license.go:90]: Correct. The change should probably just be made on the CLI to only display if you're an admin.

Comment (ericpaulsen) [reply enterprise/coderd/license/license.go:90]: @coadler i'm not sure how to build this into the CLI. can you take this on?

Comment (coadler) [reply enterprise/coderd/license/license.go:90]: Would probably be best to make an issue and prioritize into a sprint, I don't have the cycles currently.

**Pull request closed without merging**